larndsim/detsim_ep.py

```diff
@@ -413,7 +417,8 @@ def tracks_current(self, pixels, tracks, time_max, fields):
         tpc_borders_ep = ep.from_numpy(pixels, self.tpc_borders).float32()
         borders = ep.stack([tpc_borders_ep[x.astype(int)] for x in tracks_ep[:, fields.index("pixel_plane")]])
 
-        signals = ep.zeros(z_start, shape=(pixels.shape[0], pixels.shape[1], time_max))
+        signals = ep.zeros(z_start, shape=(pixels.shape[0], pixels.shape[1], time_max.astype(int).item()))
+        extras = []
```

Comment:
I think this was leftover from debugging, should be removed

Reply:
I removed line 421 `extras = []`. Is there any other residuals?

Reply:
This was the only one I saw, I think!